fk/lisjp/src/main/java/se/inera/intyg/common/lisjp/v1/model/converter/InternalToCertificate.java

```diff
@@ -681,4 +772,866 @@ private static CertificateDataValueDiagnosis createDiagnosis(String id, Diagnos
+                    .label("Resor till och från arbetet med annat färdmedel än normalt kan göra det möjligt för"
+                        + " patienten att återgå till arbetet under sjukskrivningsperioden.")
+                    .selectedText("Ja")
+                    .unselectedText("Nej")
+                    .build()
+            )
+            .value(
+                CertificateDataValueBoolean.builder()
+                    .id(ARBETSRESOR_SVAR_JSON_ID_34)
+                    .selected(internalCertificate.getArbetsresor())
+                    .build()
+            )
+            .validation(
+                new CertificateDataValidation[]{
+                    CertificateDataValidationHide.builder()
+                        .questionId(AVSTANGNING_SMITTSKYDD_SVAR_ID_27)
+                        .expression(singleExpression(AVSTANGNING_SMITTSKYDD_SVAR_JSON_ID_27))
+                        .build()
+                }
+            )
+            .build();
+    }
+
+    private static CertificateDataElement createPrognosQuestion(LisjpUtlatandeV1 internalCertificate, int index) {
+        return CertificateDataElement.builder()
+            .id(PROGNOS_SVAR_ID_39)
+            .index(index)
+            .parent(BEDOMNING_CATEGORY_ID)
+            .config(
+                CertificateDataConfigRadioMultipleCode.builder()
+                    .text("Prognos för arbetsförmåga utifrån aktuellt undersökningstillfälle")
+                    .description("En viktig information för att underlätta planeringen.")
+                    .list(
+                        Arrays.asList(
+                            RadioMultipleCode.builder()
+                                .id(PrognosTyp.MED_STOR_SANNOLIKHET.getId())
+                                .label(PrognosTyp.MED_STOR_SANNOLIKHET.getLabel())
+                                .build(),
+                            RadioMultipleCode.builder()
+                                .id(PrognosTyp.ATER_X_ANTAL_DGR.getId())
+                                .label(PrognosTyp.ATER_X_ANTAL_DGR.getLabel())
+                                .build(),
+                            RadioMultipleCode.builder()
+                                .id(PrognosTyp.SANNOLIKT_EJ_ATERGA_TILL_SYSSELSATTNING.getId())
+                                .label(PrognosTyp.SANNOLIKT_EJ_ATERGA_TILL_SYSSELSATTNING.getLabel())
+                                .build(),
+                            RadioMultipleCode.builder()
+                                .id(PrognosTyp.PROGNOS_OKLAR.getId())
+                                .label(PrognosTyp.PROGNOS_OKLAR.getLabel())
+                                .build()
+                        )
+                    )
+                    .build()
+            )
+            .value(
+                CertificateDataValueCode.builder()
+                    .id(internalCertificate.getPrognos() != null ? internalCertificate.getPrognos().getTyp().getId() : null)
```

Comment:
Consider refactoring to a private method that can create the value based if prognose is null or not.